test/k8s/service_helpers.go

```diff
@@ -900,104 +900,6 @@ func testExternalTrafficPolicyLocal(kubectl *helpers.Kubectl, ni *helpers.NodesI
 	}
 }
 
-func testHostPort(kubectl *helpers.Kubectl, ni *helpers.NodesInfo) {
```

Comment:
@cilium/sig-datapath do we have coverage of this now? I'm surprised this was unused.

Reply:
`test/controlplane/services/nodeport` now tests that we populate lbmap correctly for HostPort and https://github.com/cilium/cilium-cli/blob/master/connectivity/tests/host.go tests e2e.